testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8 import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+
+    '''Unittests for RearrangeCars class and promised fuctionality
+    '''
+
+    def setUp(self):
+
+        '''Basic setting that will be used in several tests
+        '''
+        
+        self.initial_car_order = [1, 2, 0, 3]
+        self.final_car_order = [3, 1, 2, 0]
+        self.rc = RearrangeCars(list(self.initial_car_order), list(self.final_car_order))

```

Comment:
@luxiya01  I guess that my previous version is more intuitive, since I was checking current order of car object with the final order I want (like listed below) We should not compare with self.rc_empty.final_car_order since we are not sure that final_order value inside of RearrangeCar is correct. 

`self.assertEqual(rc_empty.current_car_order, self.final_car_order)`

So in this case, additional lists are not necessary here. What do you think @luxiya01 ?

```
 _variables_to_init_state(self)
... self.current_car_order = list(self.initial_car_order)
```